dev_web.ps1

```diff
@@ -15,6 +15,9 @@ Set-ItemProperty -Path HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\
 
 #--- Tools ---
 choco install -y vscode
+code --install-extension msjsdiag.debugger-for-chrome

```

Comment:
@yodurr I "think" this might require a refreshenv in order to function.  I gave a Chocolatey presentation yesterday, and the need to refreshenv was required for me.

Reply:
In PowerShell scripts, I'm not going to recommend a "refreshenv" as it is not deterministic to know if the PowerShell profile is installed - it is more deterministic to use `Update-SessionEnvironment` after importing some of the Chocolatey module. cc @crutkas 

I'll explain at the bottom on the need for updating environment variables.